Realm/ObjectServerTests/RLMSyncTestCase.mm

```diff
@@ -67,7 +67,7 @@ @interface RLMSyncTestCase ()
 @property (nonatomic) NSTask *task;
 @end
 
-@interface RLMSyncCredentials ()
+@interface RLMAppCredentials ()
```

Comment:
This (and createAdminUserForURL) presumably aren't used any more since the debug admin provider isn't a thing?